config/routes_into_teaching.yml

```diff
@@ -45,12 +45,12 @@ routes:
         answers: ["*"]
 
   - title: Assessment only route
```

Comment:
Do we need to include `route` in the title? Makes sense in the body text but less so here?